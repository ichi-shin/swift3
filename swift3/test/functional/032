#!/bin/bash

# Virltual-Hosted-Style

. ./common

S3USER=tester

_s3_put /bucket
_s3_put /bucket/object -T /dev/null
_s3curl http://bucket.${SWIFT_HOST}/ -D - | _filter_curl xml
