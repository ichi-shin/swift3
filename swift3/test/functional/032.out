QA output created by 032
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X PUT -T /dev/null http://SWIFT_HOST/bucket/object... 200
> s3curl --id tester -- http://bucket.SWIFT_HOST/ -D -... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<ListBucketResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Name>bucket</Name>
  <Prefix/>
  <Marker/>
  <MaxKeys>1000</MaxKeys>
  <IsTruncated>false</IsTruncated>
  <Contents>
    <Key>object</Key>
    <LastModified>DATE</LastModified>
    <ETag>ETAG</ETag>
    <Size>0</Size>
    <Owner>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Owner>
    <StorageClass>STANDARD</StorageClass>
  </Contents>
</ListBucketResult>
